### PR TITLE
Add realm icon in the links-to card in overlaid header in operator mode

### PR DESCRIPTION
**Pull request by jurgenwerk**
This PR adds the realm's icon to the overlaid header of a links-to relationship. See the image below. For contrast, the contained card has a different icon (the arrow). 

image

### CODE REVIEW

File: packages/host/app/components/operator-mode/overlays.gts [hunk 1]
```diff
@@ -52,17 +62,21 @@ export default class OperatorModeOverlays extends Component<Signature> {
...
               <div class='header-title'>
-                {{#if (eq renderedCard.fieldType 'contains')}}
-                  <div class='header-icon'>
+                <div class='header-icon'>
+                  {{#if (eq renderedCard.fieldType 'contains')}}
                     {{svgJar 'icon-turn-down-right' width='22px' height='18px'}}
-                  </div>
-                {{/if}}
+                  {{else}}
+                    {{#let (load (this.getRealmInfo card)) as |result|}}
```
Comment (jurgenwerk) [inline packages/host/app/components/operator-mode/overlays.gts:72]: I found this neat `load` helper in the `ember-async-data` lib I added in this PR. It fetches async data straight from the template and it holds its own loading state.

**Comment by jurgenwerk** [comment]
A bit unrelated to the title of this PR but I snuck in another addition - a quick preview button that we will use later when we implement that functionality:

image

image

**Comment by tintinthong** [comment]
This may unrelated to the work for embedded but I see an icon and some weird behaviour now

Screenshot 2023-08-01 at 11 19 06 AM

Screenshot 2023-08-01 at 11 19 21 AM

EDIT. I tested before https://github.com/cardstack/boxel/pull/503 and I see the Pet stack above the Person (as expected). I will file a [CS-5824](https://linear.app/cardstack/issue/CS-5824/bug-wrong-stack-order-when-create-new-in-nested-contains).

**Review by tintinthong — APPROVED** [review]
The comment I made as an aside. This PR looks good

**Pull request merged**